docs/dev/connectors/pubsub.zh.md

```diff
@@ -143,22 +135,22 @@ env.addSource(pubsubSource)
 </div>
 </div>
 
-### Atleast once guarantee
+### 精确一次语义保证
 
 #### SourceFunction
 
-There are several reasons why a message might be send multiple times, such as failure scenarios on Google PubSub's side.
+有很多原因导致会一个信息会被多次发出，例如 Google PubSub 的故障。
 
-Another reason is when the acknowledgement deadline has passed. This is the time between receiving the message and between acknowledging the message. The PubSubSource will only acknowledge a message on successful checkpoints to guarantee Atleast-Once. This does mean if the time between successful checkpoints is larger than the acknowledgment deadline of your subscription messages will most likely be processed multiple times.
+另一个可能的原因是超过了确认的截止时间。这是收到信息的间隔和信息确认的间隔。PubSubSource 只有在信息被成功快照之后才会确认以保证至少一次的语义。这意味着，如果你的快照间隔大于信息确认的截止时间，那么你订阅的信息很有可能会被多次处理。
 
-For this reason it's recommended to have a (much) lower checkpoint interval than acknowledgement deadline.
+因此，我们建议把快照的间隔设置得比信息确认截止时间更短。
 
-See [PubSub](https://cloud.google.com/pubsub/docs/subscriber) for details on how to increase the acknowledgment deadline of your subscription.
+参照 [PubSub](https://cloud.google.com/pubsub/docs/subscriber) 来增加信息确认截止时间。
 
-Note: The metric `PubSubMessagesProcessedNotAcked` shows how many messages are waiting for the next checkpoint before they will be acknowledged.
+注意: `PubSubMessagesProcessedNotAcked` 显示了有多少信息正在等待下一个快照还没被确认。
```

Comment:
I think we don't need to translate "checkpoint" according to Flink Translation Specifications